lib/configuration.js

```diff
@@ -56,7 +54,7 @@ function extractServiceContext(initConfiguration) {
   };
 
   if (has(initConfiguration, 'serviceContext') &&
-      isObject(initConfiguration.serviceContext)) {
+      isPlainObject(initConfiguration.serviceContext)) {
```

Comment:
Is this change logically related to how the uncaught exception configuration works?